ts/heartbeat.ts

```diff
@@ -235,6 +235,39 @@ export class Heartbeat {
         }
       }
 
+      // should we try to improve the mesh with opportunistic grafting?
```

Comment:
```suggestion
```
This can go away, right?

Reply:
This is just a commentary about the block of code, pulled from https://github.com/libp2p/go-libp2p-pubsub/blob/8945f91465bc3ba418688684a2c29c384cd79647/gossipsub.go#L1387
I suggest we keep it